libbeat/processors/condition_test.go

```diff
@@ -93,6 +93,18 @@ func TestEqualsCondition(t *testing.T) {
 				"proc.cpu.total_p.gt": 0.5,
 			}},
 		},
+
+		{
+			Equals: &ConditionFields{fields: map[string]interface{}{
```

Comment:
I opened https://github.com/elastic/beats/pull/5960 to make adding tests cleaner. I realised when reading this that the way tests are written at the moment is not optimal for diffs.